### ISSUE TITLE
`gt` does not work on string literals

### ISSUE THREAD

**Issue by blinry** [HenryLie/svelte-i18n-lingui#6]
We tried using a line like

```js
gt("test")
```

in a TS file, and the jstsExtractor didn't catch it. We checked the code, and it seems that it currently only works on expressions like this:

```js
gt({message: "test"})
```

Would it make sense to also allow `gt` to take plain strings? Thanks!

**Comment by blinry** [comment]
Oh! We learned that

```
gt`test`
```

works! What does this syntax mean?

**Comment by HenryLie** [comment]
Hey, thanks for trying out the lib! 

I made `gt` syntax mostly the same as `$t` would in Svelte components, so it can accept either of these two forms:
- plain string with [tagged template literal syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates). In your example above: ``gt`test` `` is the right way to translate either plain strings or parameterized strings. This JS syntax behaves like calling the function called `gt` and passing in a parameterized string as its argument, so this is sort of equivalent of writing ``gt(`test`)``, just shorter :grin:
- the messageDescriptor syntax, which is like in your example `gt({message: "test"})`. This form is mostly only necessary if you'd like to add comments or context to the string extracted to the catalog, or if you'd like to use the plural form.

Let me know if you have any other question 😄

**Comment by blinry** [comment]
Makes sense! I'd suggest adding an example using `` gt`foo` `` to the README, so others will be less stumped than us! :D

Ohh, and that you can drop the parens is some some nice syntactic sugar, thanks for the explanation!
